install/envs/pc.yml

```diff
@@ -0,0 +1,25 @@
+name: donkey
+dependencies:
+
+- h5py
+- pillow
+- opencv
+- matplotlib
+- tornado
+- docopt
+- pandas
+- pylint
+- pytest
+- pip
+- imgaug
+- progress
+- moviepy
+- paho-mqtt
+- PrettyTable
+
+- pip:
+  - tensorflow==2.2.0
+  - git+https://github.com/autorope/keras-vis.git
+  - simple-pid
+  - opencv-python-headless
```

Comment:
This is a pip dependency from `imgaug` but not a Conda one (strange). Hence I explicitly added it here. I prefer to have `pip install -e .[pc]` not to pip install anything, rather all packages should be defined already here. In general Conda has better package dependency control. Without `opencv-python-headless' the example augmentation script runs, but it is very slow.

Reply:
Good to know. I don't use `conda` myself., so I am not sure about these YAML files. Thanks for fixing this.